### ISSUE TITLE
CSS parses, Less chokes on a:link

### ISSUE THREAD

**Issue by subsetpark** [less/less.ruby#20]
Here's the first two entries in my stylesheet:

```
@accent_color: #c4161c;

a:link,
a:visited {
    color:@accent_color;
    text-decoration:none;
}
```

Lessc produces 

```
!! errors were found in the .less file! 
line 3: syntax error, unexpected tSYMBEG, expecting kDO or '{' or '('
a:link,
  ^
!! errors were found in the .less file! 
line 3: syntax error, unexpected tSYMBEG, expecting kDO or '{' or '('
a:link,
  ^
```

Am I doing something wrong? I feel like I must be. But the CSS is correctly read by the server so I don't know exactly what.

**Comment by cloudhead** [comment]
Hey, good find—I'll correct this in the next patch, for now, you can keep your selectors on one line, and it should compile:

```
a:link, a:visited {
    color:@accent_color;
    text-decoration:none;
}
```

**Comment by cloudhead** [comment]
Fixed in 0.8.10
